docs/UserGuide.md

```diff
@@ -170,47 +189,75 @@ Your list of medications has been successfully shown!
 ### Daily medications:
 Displays an overview of the list of medications that you will be taking for the day.
 
-Format: `list -t today`
-* If you want to see for only a particular period of the day, simply enter `list -t today` followed by:
+Example: `meditracker> list -t today`
+* If you want to specify a particular period of the day, simply enter `list -t today` followed by:
   * `-m`: Morning (Midnight to 12 Noon)
   * `-a`: Afternoon (12pm to 6pm)
   * `-e`: Evening (6pm to Midnight)
-  * Eg: `list -t today -m` - will display the list of medications to be taken in the morning only
+  * Example: `meditracker> list -t today -m` - will display the list of medications to be taken in the morning only
```

Comment:
Is the " - " necessary to include? Perhaps removing it might look nicer.

Reply:
Implemented in  aff146c